src/controllers/BaseController.ts

```diff
@@ -51,12 +51,12 @@ export abstract class BaseController<FORM> extends BaseAsyncHttpController {
     }
 
     protected prepareViewModelFromSession(session: Session): Record<string, any> {
-        const appeal: Appeal = session
+        const applicationData: ApplicationData = session
             .getExtraData()
-            .chain<Appeal>(data => Maybe.fromNullable(data[APPEALS_KEY]))
-            .orDefault({} as Appeal);
+            .chain<ApplicationData>(data => Maybe.fromNullable(data[APPEALS_KEY]))
+            .orDefault({} as ApplicationData);
 
-        return this.prepareViewModelFromAppeal(appeal)
+        return this.prepareViewModelFromAppeal(applicationData.appeal || {})
```

Comment:
if `applicationData` is empty, would trying to access the appeal throw an error here?

Reply:
no, it would return {}

Reply:
I believe line 57 would catch that and return {}

Reply:
if applicationData is undefined, line 57 would catch it I believe and return {}, and if appeal is underfined, line 59 will catch it and return {} too